bindings/nodejs/examples/exchange/1-generate-mnemonic.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * This example generates a new random mnemonic
+ */
+
+const { AccountManager, CoinType } = require('@iota/wallet');
+
+async function run() {
+    try {
+        const manager = new AccountManager({
+            storagePath: './mnemonic-generation',
+            clientOptions: {
+                nodes: ['http://localhost:14265'],
+            },
+            coinType: CoinType.Shimmer,
```

Comment:
Add comment that IOTA coin type can be used to access staked funds

Reply:
Would be better imo to write a dedicated example for claiming shimmer.

Reply:
That could be done additionally, just want to have the comment there so it's clear